sources/packages/backend/apps/api/src/services/education-program/education-program.service.ts

```diff
@@ -513,6 +521,7 @@ export class EducationProgramService extends RecordDataModelService<EducationPro
         "assessedBy.lastName",
         "programs.assessedDate",
         "programs.effectiveEndDate",
+        "programs.isActive",
```

Comment:
Not part of the PR. The method at line 357 seems to be obsolete.

Reply:
Removing it 👍

Reply:
Method removed, I did not find any reference to it.